coalib/collecting/Collectors.py

```diff
@@ -289,12 +289,15 @@ def get_all_bears_names():
     return [bear.name for bear in get_all_bears()]
 
 
-def collect_all_bears_from_sections(sections, log_printer=None):
+def collect_all_bears_from_sections(sections,
+                                    log_printer=None,
+                                    bear_globs=tuple('**')):
     """
     Collect all kinds of bears from bear directories given in the sections.
 
     :param sections:    List of sections so bear_dirs are taken into account
     :param log_printer: Log_printer to handle logging
+    :param bear_globs:  List of patterns/bear names to match bears
```

Comment:
missing period at end.
\+
What kind of patterns? Mention that these are globs :+1: So just `List of glob patterns ...`